sdk/storage/storage-blob/src/Pipeline.ts

```diff
@@ -194,17 +195,29 @@ export function newPipeline(
   // Order is important. Closer to the API at the top & closer to the network at the bottom.
   // The credential's policy factory must appear close to the wire so it can sign any
   // changes made by other factories (like UniqueRequestIDPolicyFactory)
-  const factories: RequestPolicyFactory[] = [
-    new TelemetryPolicyFactory(pipelineOptions.telemetry),
-    new UniqueRequestIDPolicyFactory(),
-    new BrowserPolicyFactory(),
-    deserializationPolicy(), // Default deserializationPolicy is provided by protocol layer
-    new RetryPolicyFactory(pipelineOptions.retryOptions),
-    new LoggingPolicyFactory(),
-    proxyPolicy(getDefaultProxySettings((pipelineOptions.proxy || {}).url)),
-    credential
-  ];
-
+  let factories: RequestPolicyFactory[];
+  if (isBrowser()) {
```

Comment:
It's not easy to see the difference in the if else branches. how about using array concat?

```ts
factories = [ ... ];
if (isNode) {
  factories.push(proxyPolicy);
}
factories.concat([...]) // the rest